packages/core/src/App/Constants/routes-config.js

```diff
@@ -305,6 +305,38 @@ const getModules = () => {
             component: Cashier_V2,
             is_authenticated: true,
             getTitle: () => localize('Cashier'),
+            routes: [
+                {
+                    path: '/cashier-v2/deposit',
+                    component: Cashier_V2,
+                    title: 'Deposit',
+                },
+                {
+                    path: '/cashier-v2/withdrawal',
+                    component: Cashier_V2,
+                    title: 'Withdrawal',
+                },
+                {
+                    path: '/cashier-v2/payment-agent',
+                    component: Cashier_V2,
+                    title: 'Payment agents',
```

Comment:
I guess we need to define all the routes here again just due to how whole core works?

Reply:
hey @wojciech-deriv 
it was done to handle 404 page properly
but
i removed the config from the core and implemented 404 page check inside cashier-v2 package

Reply:
sorted